### PR TITLE
aksarc 0.2.0b1 VMware support release

### PR DESCRIPTION
**Pull request by TheOnlyWei**
### Related command
az aksarc
https://learn.microsoft.com/en-us/cli/azure/aksarc?view=azure-cli-latest

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
**Note**: `azdev style <YOUR_EXT>` is not enforced currently for our extension in our private repository.
- [x] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [x] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

### About Extension Publish
**Note that we are releasing an extension from a private repository and need to manually update the `src/index.json`!**

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

**Comment by TheOnlyWei** [comment]
`Verify Extensions Index` failed due to someone else's extension `spring-1.20.1-py3-none-any.whl`.

**Comment by wangzelin007** [comment]
> `Verify Extensions Index` pipeline failed due to someone else's extension called `spring-1.20.1-py3-none-any.whl`. It should be good to merge.

It will be fixed by this [PR](https://github.com/Azure/azure-cli-extensions/pull/7410)

File: src/index.json [hunk 1]
```diff
@@ -10300,6 +10300,60 @@
             }
         ],
         "aksarc": [
+            {
```
Comment (jsntcy) [inline src/index.json:10303]: We suggest you change the version to `1.0.0b1` (preview) or `1.0.0` (stable).

As now CLI extension version should conform to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

Comment (TheOnlyWei) [reply src/index.json:10303]: What is the reason for this? We are releasing an older version of our extension. Are you saying the minimum version has to be 1.0.0?

**Comment by TheOnlyWei** [comment]
Closing this in favor of this updated PR:
https://github.com/Azure/azure-cli-extensions/pull/7416

**Pull request closed without merging**